docs/index.md

```diff
@@ -17,6 +17,8 @@ OTEAPI Core provides the core functionality of OTEAPI, which stands for the *Ope
 
 It uses the [strategy](https://en.wikipedia.org/wiki/Strategy_pattern) software design pattern to implement a simple and easy to extend access to a large range of data resources.
 Semantic interoperability is supported via mapping of data models describing the data to ontologies.
+We provide a set of abstract interfaces for strategies and data models used in their configuration.
```

Comment:
I do not think that we are supposed to make the changes in the docs part?

Reply:
I'm not sure, the tests fail unless I update it in both places...

Reply:
Yes, I think you have to update both places, but pre-commit will update the docs for you.

If you aren't using pre-commit, install it with

    pip install pre-commit
    pre-commit install